nhs/data/handling.py

```diff
@@ -148,17 +148,20 @@ def standardize_names(
 
     Examples
     --------
-    >>> def standardize_names(
-    ...     {
-                "name1": <LazyFrame>,
-    ...         "name2": <LazyFrame>},
-    ...         "name3" <LazyFrame>,
-    ...     },
-    ...     "identification",
-    ...     "abbreviation_column_name",
-    ...     "Long_column_name"
-    ... )
-    {"file_identify1_with_expanded_names": <LazyFrame>, "file_identify2_with_expanded_names": <LazyFrame>}
+    >>> frames = {
+    ...     "file_identify1": pl.LazyFrame({"SHORT1": [1, 2, 3], "SHORT2": [4, 5, 6]})
+    ...     "file_identify2": pl.LazyFrame({"SHORT1": [1, 2, 3], "SHORT2": [4, 5, 6]})
+    ... }
+    >>> metadata = pl.LazyFrame({
+    ...     "file_names": ["file_identify1"],
+    ...     "abbreviated": ["SHORT1", "SHORT2"],
+    ...     "unabbreviated": ["LONG 1", "LONG 2"]
+    ... })
+    >>> frames_out = standardize_names(dfs, metadata, "file_names", "abbreviated", "unabbreviated")
+    >>> frames_out["file_identify1"].columns
+    ["long_1", "long_2"]
+    >>> frames_out["file_identify2"].columns # No change, name not in `metadata["file_names"]`
+    ["SHORT1", "SHORT2"]
```

Comment:
Your example also didn't make sense, we are not changing the keys in `df_dict`, we are changing the column names. You should have examples more akin to above: we define 2 `LazyFrame`s with some column name `SHORT1` and `SHORT2`, and run it through the function. We then see how the column names have been changed.